src/IoTeX/Staking.cpp

```diff
@@ -0,0 +1,211 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Staking.h"
+
+using namespace TW;
+
+namespace TW::IoTeX {
+
+/// Pack uint64 to uint256 in big-endian order
+///
+Data packUint64ToUint256(uint64_t value)
+{
+    Data result(32);
+
+    for (auto it = result.rbegin(); it < result.rend(); it++) {
+        *it = (byte)value;
+        if ((value = value >> 8) == 0)
+            break;
+    }
+    return result;
+}
+
+/// Pack byte array to end of calldata
+///
+/// \param dataLen length of the last input _data
+/// \param _data payload bytes that user wants to write
+///
+Data packBytesToEnd(uint64_t offset, uint64_t dataLen, const byte *_data)
```

Comment:
- Some parameters are prefixed by _ underscore, others not.  What is the meaning? Suggest leaving out prefixes.

Reply:
ok

Reply:
please check latest 2 commits